File: Core/Components/InputLike.cs
using Bridge.Html5;
using Core.Models;

namespace Core.Components
{
    public class Textarea : Textbox
    {
        public Textarea(Component ui, HTMLElement ele = null) : base(ui, ele)
        {
            MultipleLine = true;
        }
    }

    public class Password : Textbox
    {
        public Password(Component ui, HTMLElement ele = null) : base(ui, ele)
        {
            Password = true;
        }
    }
}

File: CoreAPI/wwwRoot/js/core/InputLike.js
import Textbox from './textbox.js';


class Textarea extends Textbox {
     /**
     * @param {Component} ui
     * @param {HTMLElement} [ele=null] 
     */
    constructor(ui, ele = null) {
        super(ui, ele);
        this.MultipleLine = true;
    }
}

class Password extends Textbox {
     /**
     * @param {Component} ui
     * @param {HTMLElement} [ele=null] 
     */
    constructor(ui, ele = null) {
        super(ui, ele);
        this.Password = true;
    }
}

window.Core2 = window.Core || {};
window.Core2.Textarea = Textarea;
window.Core2.Password = Password;

export { Textarea, Password };
